README.md

```diff
@@ -148,7 +148,7 @@ from hydra_agent.agent import Agent
 agent = Agent("http://localhost:8080/serverapi")
 agent.get("http://localhost:8080/serverapi/DroneCollection/123-123-123-123")
 ```
-
+**Remember that it's important you use an hydrus updated version, currently the one under the develop branch**
```

Comment:
we want probably to support only the master branch in hydrus, anytime the agent needs code merged in hydrus, just notify the mentors. I opened a PR for merging https://github.com/HTTP-APIs/hydrus/pull/405